en/docs/deploy/deploy-to-devant.md

```diff
@@ -2,6 +2,23 @@
 [Devant](https://console.devant.dev/) is a powerful IPaaS with first-class AI support. Incorporate AI agents into the integrations you build in low-code and pro-code, and move away from siloed systems to intelligent digital experiences with Devant by WSO2—the AI iPaaS that your AI Agents can call “home”.
 WSO2 Integrator: BI provides a seamless integration experience with Devant. You can deploy your integrations to Devant with just a few clicks.
 
-Refer to the [Devant quick start guide](https://wso2.com/devant/docs/quick-start-guides/schedule-your-first-automation/) to deploy your integration in Devant. This guide provides step-by-step instructions to help you set up, configure, and successfully execute your integration with minimal effort.
+1. Login to Devant
```

Comment:
Isn't this bit misleading, as we are not opening and logging in to Devant Console here?